R/install.R

```diff
@@ -81,7 +83,13 @@ install_cmdstan <- function(dir = NULL,
                             version = NULL,
                             release_url = NULL,
                             cpp_options = list(),
-                            check_toolchain = TRUE) {
+                            check_toolchain = TRUE,
+                            wsl = FALSE) {
```

Comment:
Should we give a warning on non-Windows OSes if `wsl = TRUE`?

Reply:
Actually, I think this should stop in case of `wsl = TRUE` and the OS not being Windows. I just tried it on my Mac and it started installing in `wsl-cmdstan-2.30.0`.